traffic_ops/traffic_ops_golang/deliveryservice/servers/servers.go

```diff
@@ -896,7 +896,19 @@ func (dss *TODSSDeliveryService) Read(h http.Header, useIMS bool) ([]interface{}
 		where = "WHERE "
 	}
 
-	where += "ds.id in (SELECT deliveryService FROM deliveryservice_server WHERE server = :server) OR ds.id in (SELECT id FROM deliveryservice WHERE topology in ( SELECT topology FROM topology_cachegroup WHERE cachegroup = ( SELECT name FROM cachegroup WHERE id = ( SELECT cachegroup FROM server WHERE id = :server ))))"
+	where += `
+		ds.id in (
+			SELECT deliveryService FROM deliveryservice_server WHERE server = :server
+		) OR ds.id in (
+			SELECT id FROM deliveryservice 
+			WHERE topology in ( 
+				SELECT topology FROM topology_cachegroup 
+				WHERE cachegroup = ( 
+					SELECT name FROM cachegroup 
+					WHERE id = ( 
+						SELECT cachegroup FROM server WHERE id = :server 
+					))))
```

Comment:
Queries should not be indented like the rest of the function, because if you look at the complete, assembled query that Traffic Ops logs, that indentation does not make sense.

So, the query fragment should be unindented by 2 tabs.

Reply:
Fixed it.